internal/compiler/diagnostics.rs

```diff
@@ -285,6 +285,11 @@ impl Diagnostic {
     pub fn source_file(&self) -> Option<&Path> {
         self.span.source_file().map(|sf| sf.path())
     }
+
+    /// return the path of the source file where this error is attached
+    pub fn source_version(&self) -> Option<i32> {
```

Comment:
See the note at the beginning of the `impl` 
This is re-export as public API in slint_interpreter.

Does it really make sense to expose this?

Reply:
How would you extract that info without this helper? Everything in that struct is private, so I think I will need an accessor to get the information out.

It should return a SourceFileVersion though.